partiql-eval/src/test/kotlin/org/partiql/lang/eval/EvaluatingCompilerExcludeTests.kt

```diff
@@ -810,265 +777,8 @@ class EvaluatingCompilerExcludeTests : EvaluatorTestBase() {
 
     @ParameterizedTest
     @ArgumentsSource(ExcludeTests::class)
-    fun validExcludeTests(tc: EvaluatorTestCase) = testHarness.runEvaluatorTestCase(
+    fun validExcludeTests(tc: EvaluatorTestCase) = runEvaluatorTestCase(
         tc,
         EvaluationSession.standard()
     )
-// TODO: subsumption tests to be added back in https://github.com/partiql/partiql-lang-kotlin/pull/1280
```

Comment:
Subsumption tests moved to `partiql-eval/src/test/kotlin/org/partiql/lang/eval/internal/exclude/CompiledExcludeExprTest.kt`